lib/seattleflu/id3c/cli/command/etl/redcap_det_scan.py

```diff
@@ -93,6 +93,19 @@ def command_for_each_project(function):
 
 @command_for_each_project
 def redcap_det_scan(*, db: DatabaseSession, cache: TTLCache, det: dict, redcap_record: REDCapRecord) -> Optional[dict]:
+    # Skip record if the illness_questionnaire is not complete, because this is
+    # a "false" enrollment where the participant was not mailed a swab kit.
+    # We must verify illness_questionnaire with the `illness_q_date` field
+    # since there is a bug in REDCap that sometimes leaves the questionnaire marked incomplete/unverified.
+    # We must have another check of the back_end_mail_scans because sometimes
+    # the `illness_q_date` field is not filled in due to a bug in REDCap.
+    # By verifying illness_questionnaire is complete first, we minimize the
+    # delay in data ingestion since the back_end_mail_scans is completed the day after enrollment.
+    #   -Jover, 29 June 2020
+    if not redcap_record['illness_q_date'] and redcap_record['back_end_mail_scans_complete'] != '2':
```

Comment:
Instead of using the magic value of `2`, let's use `id3c.cli.redcap.is_complete()`?

https://github.com/seattleflu/id3c/blob/f401e64480424821d39988940f7c1cd71ea94209/lib/id3c/cli/redcap.py#L244-L265

Reply:
Ah, yup. Thank you!